### PR TITLE
Show remote mic activity

### PR DESCRIPTION
**Pull request by niuyunsi**

### CODE REVIEW

File: docs/src/components/App.tsx [hunk 1]
```diff
@@ -37,8 +39,17 @@ export class App extends React.Component<{}, {}> {
     }
 
     private renderRemoteStream(stream: ConferenceStream) {
+        // NOTE(yunsi): Use the fisrt 10 characters as the remote name
+        // TOEO(yunsi): Find a better way to define remote name
```
Comment (linweigao) [inline docs/src/components/App.tsx]: typo => TODO

File: docs/src/components/App.tsx [hunk 2]
```diff
@@ -37,8 +39,17 @@ export class App extends React.Component<{}, {}> {
...
+        const name = stream.id.substring(0, 10);
```
Comment (linweigao) [inline docs/src/components/App.tsx]: What kind of name need to display?

File: docs/src/index.scss [hunk 3]
```diff
@@ -35,15 +35,31 @@
...
-        video {
-            width: 100%;
-            height: auto;
+
+        &__name {
+            position: absolute;
+            bottom: 15px;
+            left: 15px;
+
+            &--is-speaking {
```
Comment (linweigao) [inline docs/src/index.scss:57]: isSpeaking?

Comment (niuyunsi) [reply docs/src/index.scss:57]: https://en.bem.info/methodology/naming-convention/
Names of BEM entities are written using numbers and lowercase Latin characters. Individual words within names are separated by a hyphen (-).

File: docs/src/components/App.tsx (hunk 2 above)
Comment (niuyunsi) [reply docs/src/components/App.tsx]: In spreed every remote video has their user name displayed at the bottom of the video, the user name will turn green if that user is speaking.

File: docs/src/components/App.tsx (hunk 1 above)
Comment (niuyunsi) [reply docs/src/components/App.tsx]: DONE

**Review by asinner — APPROVED** [review]

**Review by linweigao — APPROVED** [review]

**Pull request merged**